File: src/main/scala/com/bombproj/vo/DataTableIndexesVO.java
package com.bombproj.vo;

import lombok.Data;

@Data
public class DataTableIndexesVO {

    private String id;
    private String indexName;
    private String[] fieldNames;
    private String indexType;
    private String indexMethod;
    private Integer state;
    private String version;
    private String createUserName;

}

File: src/main/scala/com/bombproj/vo/DataTableIndexesVO.scala
package com.bombproj.vo

import scala.beans.BeanProperty

case class DataTableIndexesVO() {

    @BeanProperty var id: String = _
    @BeanProperty var indexName: String = _
    @BeanProperty var fieldNames: Array[String] = _
    @BeanProperty var indexType: String = _
    @BeanProperty var indexMethod: String = _
    @BeanProperty var state: Int = _
    @BeanProperty var version: String = _
    @BeanProperty var createUserName: String = _

}
